### PR TITLE
Features/spec compliance

### PR DESCRIPTION
**Pull request by loudmouth**
~Do not merge until snowplow events show up in dashboard~

#### Adds
- Crash reporting
- Snowplow tracking for App Store builds
- Modal error for deep link routing to non-existent course and/or lesson

#### Changes
- Url parameter for enabling editorial featuers

### CODE REVIEW

**Review by Roblinde — APPROVED** [review]

File: the-example-app.swift/Info.plist [hunk 1]
```diff
@@ -37,6 +37,20 @@
 	<string>$(CONTENTFUL_PREVIEW_TOKEN)</string>
 	<key>CONTENTFUL_SPACE_ID</key>
 	<string>$(CONTENTFUL_SPACE_ID)</string>
+	<key>Fabric</key>
+	<dict>
+		<key>APIKey</key>
+		<string>8a9ee3680cf2a847e9a1be43cc40a1b9deed2047</string>
```
Comment (mariobodemann) [inline the-example-app.swift/Info.plist:43]: Out of curiosity: Is it wise to store the key in plain text here?

**Review by mariobodemann — COMMENTED** [review]
Looks good to me, just a quick question about api keys ... :D

**Comment by loudmouth** [comment]
@mariobodemann i only have one other option which is to use [cocoapod-keys](https://github.com/orta/cocoapods-keys) to obsfucate and scramble the keys—however, if i do that it means that a `pod install` will fail for any person cloning the repo as they will need to create a parallel project in crashlytics and input their own API key.

So basically, I think this is the best option...

**Comment by loudmouth** [comment]
I can confirm that events show up in snowplow

**Pull request merged**